autoblocks/_impl/testing/models.py

```diff
@@ -13,6 +31,26 @@ class Threshold:
     gte: Optional[float] = None
 
 
+@dataclasses.dataclass()
+class EventEvaluation:
+    evaluator_external_id: str
+    score: float
+    id: Optional[str] = dataclasses.field(default_factory=lambda: str(uuid.uuid4()))
+    metadata: Optional[dict] = None
+    threshold: Optional[Threshold] = None
+
+    @classmethod
+    def to_json(cls, event_evaluation):
+        return dict(
+            evaluatorExternalId=event_evaluation.evaluator_external_id,
+            id=str(event_evaluation.id),
+            score=event_evaluation.score,
+            metadata=dict(event_evaluation.metadata) if event_evaluation.metadata else None,
```

Comment:
i think you can just pass `event_evaluation.metadata` directly? isn't metadata just `Optional[dict]`?